src/components/UserDashboard/SavedQueries/index.js

```diff
@@ -169,70 +166,83 @@ class SavedQueries extends React.Component {
             );
           })}
         </Box>
-      </Scroll>
+      </>
     ) : (
-      <Scroll key="files-queries">
-        <Box mt={2} mb={2}>
-          {fileQueries
-            .filter((q) => q.alias && q.content.Files)
-            .map((q) => ({
-              ...q,
-              date: Number(new Date(q.creationDate)),
-              link: `/search${q.content.longUrl.split('/search')[1]}`,
-            }))
-            .slice()
-            .sort((a, b) => b.date - a.date)
-            .map((q) => (
-              <QueryBlock key={q.id} query={q} inactive={deletingIds.includes(q.id)} />
-            ))}
-        </Box>
-      </Scroll>
+      <Box mt={2} mb={2}>
+        {fileQueries
+          .filter((q) => q.alias && q.content.Files)
+          .map((q) => ({
+            ...q,
+            date: Number(new Date(q.creationDate)),
+            link: `/search${q.content.longUrl.split('/search')[1]}`,
+          }))
+          .slice()
+          .sort((a, b) => b.date - a.date)
+          .map((q) => (
+            <QueryBlock key={q.id} query={q} inactive={deletingIds.includes(q.id)} />
+          ))}
+      </Box>
     );
   };
 
+  onTabChange = (key) => {
+    this.setState({ activeTab: key });
+  };
+
   render() {
     const {
       state: { queries: fileQueries, loadingQueries },
       virtualStudies,
     } = this.props;
 
+    const tabList = [
```

Comment:
can this be extracted in a class method? such as generateTabList

Reply:
im not sure a method is required... a list is just fine. There is only two possible tabs anyway.